### DISCUSSION TITLE
3.1.4. TSML UI Widgets

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#768] [Q&A]
In the previous version, the widget for Meeting Top only appeared on the list page. It did not show up in the meeting detail. 

In this version it shows up on both places.

I could easily type up some words and put it on a page using the shortcode [tsml_react] and forget about the widget and that also shows up in both places. I was hoping the widget would not be on the meeting detail. 

The meeting detail bottom does not appear on either the list page or the detail page.

**Comment by joshreisner** [comment]
TSML team: perhaps we should create an issue or reopen #686 ?

**Reply by brianw-area24** [reply]
Reopened #686
